src/events/deposits.upsert.service.ts

```diff
@@ -151,4 +151,96 @@ export class DepositsUpsertService {
     }
     return result[0].count;
   }
+
+  async enqueueFixMismatchedDeposits(): Promise<void> {
+    await this.graphileWorkerService.addJob(
+      GraphileWorkerPattern.FIX_MISMATCHED_DEPOSITS,
+    );
+  }
```

Comment:
nit: I think we can move this directly into the controller, like we do for refresh_points. I don't think we need this method.